flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -568,6 +585,49 @@ under the License.
 						</configuration>
 					</execution>
 
+					<execution>
+						<id>fat-jar-kafka-example</id>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<shadeTestJar>false</shadeTestJar>
+							<shadedArtifactAttached>false</shadedArtifactAttached>
+							<createDependencyReducedPom>false</createDependencyReducedPom>
+							<transformers>
+								<transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+									<mainClass>org.apache.flink.streaming.examples.kafka.KafkaExample</mainClass>
+								</transformer>
+							</transformers>
+							<finalName>KafkaExample</finalName>
+							<!-- <outputFile>Kafka.jar</outputFile> -->
```

Comment:
Why do we need this comment?

Reply:
It comes from the execution of `fat-jar-kafka-010-example`, I just copied it. Will remove.